eng/pipelines/templates/stages/archetype-cpp-release.yml

```
@@ -69,6 +69,67 @@ stages:
                           # we override the regular script path because we have cloned the build tools repo as a separate artifact.
                           ScriptPath: '$(Build.SourcesDirectory)/eng/common/scripts/copy-docs-to-blobstorage.ps1'
 
+          - ${{ if ne(artifact.options.skipPublishVcpkg, 'true') }}:
```

```suggestion
          - ${{ if ne(artifact.skipPublishVcpkg, 'true') }}:
```

We recently removed options from all the other places.